src/Format/Ini/Value.cpp

```diff
@@ -76,7 +76,7 @@ int Value::toInt() const
     {
         return std::stoi(_value);
     }
-    catch (std::invalid_argument)
```

Comment:
That is a common practice, I see a lot of code with that approach from different developers.
But this is quite strange for me =)